vmms/localDocker.py

```diff
@@ -135,7 +135,7 @@ def runJob(self, vm, runTimeout, maxOutputFileSize):
         volumePath = self.getVolumePath(instanceName)
         args = ['docker', 'run', '--name', instanceName, '-v']
         args = args + ['%s:%s' % (volumePath, '/home/mount')]
-        args = args + ['-m %dM', config.Config.VM_ULIMIT_USER_RAM]
+        args = args + ['-m%dM' % (config.Config.VM_ULIMIT_USER_RAM)]
```

Comment:
Why remove the space between the flag `-m` and the parameter?

Reply:
if you leave the space in the docker container will not start it reads the number as the number with a space in front and is not happy this took me a long time to figure out.

Reply:
hmm i tested it locally and it works either way. may be a python thing